fedora/Dockerfile.fedora29

```diff
@@ -0,0 +1,23 @@
+FROM fedora:29
```

Comment:
The build of Fedora 29 brokes with dependency exclusion:

``` bash
--2019-05-22 14:12:11--  https://raw.githubusercontent.com/freeipa/freeipa/master/freeipa.spec.in
Resolving raw.githubusercontent.com (raw.githubusercontent.com)... 151.101.92.133
Connecting to raw.githubusercontent.com (raw.githubusercontent.com)|151.101.92.133|:443... connected.
HTTP request sent, awaiting response... 200 OK
Length: 47828 (47K) [text/plain]
Saving to: 'freeipa.spec.in'

     0K .......... .......... .......... .......... ......    100% 5.16M=0.009s

2019-05-22 14:12:12 (5.16 MB/s) - 'freeipa.spec.in' saved [47828/47828]

Last metadata expiration check: 0:00:50 ago on Wed May 22 14:11:23 2019.
Package gcc-8.3.1-2.fc29.x86_64 is already installed.
Package gettext-0.19.8.1-18.fc29.x86_64 is already installed.
Package make-1:4.2.1-10.fc29.x86_64 is already installed.
Package pkgconf-pkg-config-1.5.3-2.fc29.x86_64 is already installed.
Package python3-dateutil-1:2.7.5-1.fc29.noarch is already installed.
Package python3-setuptools-40.4.3-1.fc29.noarch is already installed.
Package python3-six-1.11.0-6.fc29.noarch is already installed.
Package systemd-239-12.git8bca462.fc29.x86_64 is already installed.
Error:
 Problem: cannot install the best candidate for the job
  - package nodejs-1:10.15.3-2.module_f29+4050+4146ff7f.x86_64 is excluded
(try to add '--skip-broken' to skip uninstallable packages)
The command '/bin/sh -c dnf install -y @buildsys-build @development-tools     && wget https://raw.githubusercontent.com/freeipa/freeipa/master/freeipa.spec.in     && dnf builddep -y --spec ./freeipa.spec.in         -D "with_lint 1" -D "with_wheels 1" --best --allowerasing     && dnf clean all     && rm -f ./freeipa.spec.in' returned a non-zero code: 1
```

Reply:
This is also occurring to Fedora 30 after https://github.com/freeipa/freeipa-builder/pull/4#discussion_r286748841.

Reply:
I tried to disable `fedora-modular` and `updates-modular` repos to see what would happen, but I've hit the same error.

Reply:
Problem solved